packages/core/src/elements/Checkbox/FeCheckbox.scss

```diff
@@ -33,6 +33,7 @@
 
   .fe-checkbox__label {
     vertical-align: middle;
+    margin: 0;
   }
```

Comment:
Why you set the margin to 0 in the FeCheckbox?

Reply:
In the task [FR-1465](https://frontegg.atlassian.net/browse/FR-1465) I must fix the alignment for checkboxes.
But the problem on the live site:
![Снимок экрана 2021-01-18 в 10 12 27](https://user-images.githubusercontent.com/4667761/104889125-662a6700-5976-11eb-8ea2-9d3414f5d60b.png)
I have decided that it will better when changing it on the live site.